src/workerd/io/worker.c++

```diff
@@ -2424,6 +2424,13 @@ private:
     }
 
     kj::Promise<void> outgoingLoop() {
+      // Pause before sending outgoing messages when a connection is first received. V8 starts
+      // sending messages as soon as it sees there is an inspector client. The process at the
+      // other end of the pipe may not be ready for messages right away (it looks like Chrome does
+      // not render CDP messages, and responses, exchanged before the inspector window is rendered).
+      // (Bug: https://github.com/cloudflare/workerd/issues/1201, needs some more investigation).
```

Comment:
Can you expand this to explain why 300 milliseconds was chosen? Is it an arbitrary number, are there any risks that this number might suddenly no longer be adequate as some point?

Reply:
Thank you!

I'll update the comment. It is arbitrary: I tried 1s, that worked, tried 300ms and that worked. To the eye, the 300ms is not noticed, the user clicks to pop-up an inspector window and the messages are then rendered into them. Tested on my Linux box, just about to try an MBP.

My working guess is that the problem is that the user clicks to pop-up an inspector window, the initial set of messages are exchanged before the inspector window is rendered, then the inspector window is rendered, but it doesn't render messages that have already come-and-gone. This affects chrome devtools and our devtools.